smartmin/views.py

```diff
@@ -141,10 +141,6 @@ def lookup_obj_attribute(self, obj, field):
         # next up is the object itself
         obj_field = getattr(obj, curr_field, None)
 
-        # if it is callable, do so
-        if obj_field and getattr(obj_field, '__call__', None):
```

Comment:
SmartReadView defaults to model._meta.fields but I just checked and that only includes concrete fields so I guess we can treat putting a m2m in `fields` of the view as a developer error. I'll put the calling code back in as is.